lib/src/generator/template_data.dart

```diff
@@ -217,12 +229,21 @@ class LibraryTemplateData extends TemplateData<Library>
 
   @override
   Library get self => library;
+
+  @override
+  String? get aboveSidebarPath => self.aboveSidebarPath;
+  @override
+  String? get belowSidebarPath => self.belowSidebarPath;
 }
 
 /// Template data for Mixin declarations.
 class MixinTemplateData extends InheritingContainerTemplateData<Mixin> {
-  MixinTemplateData(super.htmlOptions, super.packageGraph, super.library,
-      super.mixin, super.sidebarForLibrary, super.sidebarForContainer);
+  MixinTemplateData(
```

Comment:
These constructors no longer need to take `sidebarForLibrary` and `sidebarForContainer`, the previously computed-and-cached HTML blobs.